packages/router/src/link.tsx

```diff
@@ -6,16 +6,18 @@
 import React, { forwardRef } from 'react'
 
 import { navigate } from './history'
+import type { NavigateOptions } from './history'
 
 export interface LinkProps {
   to: string
   onClick?: React.MouseEventHandler<HTMLAnchorElement>
+  options?: NavigateOptions
```

Comment:
The replace option can actually be used to remedy the fact that the `back()` can't be blocked by replacing the top item twice instead of once. This is the only component which doesn't implement that yet. (even the routes do) and I left it there after testing that workaround.

Reply:
So it's only vaguely related to the 'functionality'. should I seperate it?

Reply:
Interesting with that workaround. Might be worth adding to the docs. What do you think?

If you do add it to the docs that'd nicely tie the change together with the other changes in this PR, and you could keep everything in this one PR 🙂 It'd also be a worthwhile addition to the Playwright test

Reply:
I wouldn't really suggest it.. the thing is with that workaround it doesn't really lock you on a route .. it really navigates back first and then goes back to the original route .. so the state of a form would be lost. although there may be a use for it.. but if you lose the state anyway.. allowing back() and just disabling navigation in the UI is the better UX option imo.

Reply:
After writing this all out I'd say it makes sense to seperate it. no need to couple the changes.